tools/python/python_bootstrap_template.txt

```diff
@@ -354,11 +355,20 @@ def ExecuteFile(python_program, main_filename, args, env, module_space,
     ret_code = _RunForCoverage(python_program, main_filename, args, env,
                                coverage_entrypoint, workspace)
   else:
-    ret_code = subprocess.call(
-      [python_program, main_filename] + args,
-      env=env,
-      cwd=workspace
-    )
+    # Temporarily ignoring SIGINT to not interfere with python_program's own
```

Comment:
Assuming this is right, perhaps the same should be applied to `_RunForCoverage`. I'm not sure yet.